### PR TITLE
fix projector topic. Agenda.item_number is optional

### PR DESCRIPTION
**Pull request by ostcar**
See #278

### CODE REVIEW

File: internal/projector/slide/topic.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func Topic(store *projector.SlideStore) {
 
 		var itemNumber string
 		if topic.AgendaItemID > 0 {
-			itemNumber = datastore.String(ctx, fetch.Fetch, "agenda_item/%d/item_number", topic.AgendaItemID)
+			itemNumber = datastore.String(ctx, fetch.FetchIfExist, "agenda_item/%d/item_number", topic.AgendaItemID)
```
Comment (r-peschke) [inline internal/projector/slide/topic.go:51]: fetch.Fetch should be sufficient. There should be an error, if there is no object at all for the given AgendaItemID, but not, if the field item_number is empty. There are numerous cases this way.

Comment (ostcar) [reply internal/projector/slide/topic.go:51]: The autoupdate service works on keys/fqfields. There is no difference if an element or a field does not exist. If your code want's to check if an element exists, you have to fetch element/ID/id.

If there are numerous cases where this is wrong, all of them have to be fixed.

If the bug from Finn does still exist, then please tell me how to reproduce it.

Comment (ostcar) [reply internal/projector/slide/topic.go:51]: If you want, we could add a method fetch.FetchIfElemetExist that also fetches the id field and only skips the error of the id does exist but the other field does not.

**Review by r-peschke — APPROVED** [review]

**Comment by r-peschke** [comment]
Approved and merged as template for more fields like this

**Pull request merged**